serialization/json-ld.md

```diff
@@ -0,0 +1,12 @@
+# JSON-LD serialization
+
+## JSON-LD context file
+The context file includes every profile name, class name, property name, 
+and vocabulary name from all profiles, as well as the "xsd" namespace and maps them to their respective URIs.
```

Comment:
This info ("includes every...") is not enough (nor is it correct).

Based on what rule `"locator"` is `"core:locator"` and not `"security:locator"` ?

Reply:
Good point, I updated the description. There is still an open question on how we handle duplicated property names that are not part of Core. Which namespace wins? No one?